polaris/management/commands/testnet.py

```diff
@@ -320,3 +417,162 @@ def get_balance(self, code, issuer_public_key, json) -> Optional[str]:
+        envelope.sign(distribution)
+        try:
+            self.server.submit_transaction(envelope)
+        except BaseHorizonError as e:
+            print(
+                f"Failed to create trustline "
+                f"Result codes: {e.extras.get('result_codes')}"
+            )
+        else:
+            print(f"Success! Trustline to {code}:{asset_issuer_public_key} created!")
+
+    def add_asset_to_db(self, **options):
+        """
+        Add an asset to the database
+        """
+        asset_code = options.get("asset")
+        issuer = options.get("issuer_public")
+        distribution_seed = options.get("distribution_seed")
+        if not (issuer and asset_code):
+            print(f"\ninvalid value in options {options}")
+            return
+        for asset in Asset.objects.filter(issuer__isnull=False):
+            if asset.code == asset_code:
+                print(f"\nAsset {asset_code}:{issuer} already exists in the database, skipping...")
+                return
+        
+        print(f"\nAdding asset {asset_code}:{issuer} to database")
+        Asset.objects.create(
+            code=asset_code,
+            issuer=issuer,
+            distribution_seed=distribution_seed,
+            sep24_enabled=options.get("sep24_enabled") or True,
+            deposit_enabled=options.get("deposit_enabled") or True,
+            withdrawal_enabled=options.get("withdrawal_enabled") or True,
+            symbol="$"
+        )
+        print(f"Asset {asset_code}:{issuer} added to database")
+    
+    def delete_asset_from_db(self, **options):
+        """
+        Delete an asset from the database
+        """
+        asset_code = options.get("asset")
+        issuer = options.get("issuer_public")
+        
+        asset_to_delete = Asset.objects.filter(code=asset_code, issuer=issuer).first()
+        if not asset_to_delete:
+            print("\nNo asset matching {asset_code}:{issuer} found to delete")
+            return
+        
+        asset_to_delete.delete()
+        print(f"\nAsset: {asset_code}:{issuer} deleted from the database")
+        
+    def get_largest_circle_wallet(self, api_key: str):
+        """
+        Get the Circle testnet wallet with the largest USD balance
+        """
+        url = "https://api-sandbox.circle.com/v1/wallets"
+
+        headers = {
+            "accept": "application/json",
+            "authorization": f"Bearer {api_key}"
+        }
+
+        response = requests.get(url, headers=headers)
+        wallets = response.json().get("data")
+        max_wallet_balance = ("", float(0))  
+        for wallet in wallets:
+            for balance in wallet["balances"]:
+                if balance["currency"] == "USD" and float(balance["amount"]) > max_wallet_balance[1]:
+                    max_wallet_balance = (wallet["walletId"], float(balance["amount"]))
+        print(f"\nLargest circle wallet: {max_wallet_balance}")
+        return max_wallet_balance
```

Comment:
I don't think we should have this implicit rule where the wallet with the largest balance is used. Instead we should require the dev to pass the wallet ID. If we don't want to do that, we should default to the master wallet ID and have the dev confirm before we use it.